gax-httpjson/src/main/java/com/google/api/gax/httpjson/HttpJsonCallContext.java

```diff
@@ -207,4 +242,30 @@ public boolean equals(Object o) {
   public int hashCode() {
     return Objects.hash();
   }
+
+  private static ImmutableMap<String, List<String>> mergeExtraHeaders(
+      Map<String, List<String>> oldExtraHeaders, Map<String, List<String>> newExtraHeaders) {
+    ImmutableMap.Builder<String, List<String>> builder =
```

Comment:
This is a copy-paste from grpc version. Is there absolutely no non-ugly way to avoid this code duplication? I know that httpjson package already has similar code duplication blocks, but we should still try to avoid it.